src/pages/yonghyun/Main/Comments/Feed.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import DG from './DG';
+import '../Main.scss';
+
+class Feed extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      like: false,
+      content: '',
+      userName: '4.21ee',
+      replList: [],
+      id: 5,
+    };
+  }
```

Comment:
- constructor 에서 바로 replList의 초기값을 넣어주실 수 있습니다.
```suggestion
  constructor(props) {
    super(props);
    this.state = {
      like: false,
      content: '',
      userName: '4.21ee',
      replList: this.props.comment,
      id: 5,
    };
  }
```